doc/ug/getting_started_sw.md

```diff
@@ -24,6 +24,14 @@ $ ninja -C build-out/sw/fpga sw/device/examples/hello_world/hello_world_export
 $ ninja -C build-out/sw/fpga all
 ```
 
+If your RISC-V toolchain isn't located in the default `/tools/riscv` location you use the `TOOLCHAIN_PATH` environment variable to set a different location before running `meson_init.sh`:
+
+```console
+# Set toolchain location
+$ export TOOLCHAIN_PATH=/path/to/toolchain
```

Comment:
Personally, I would write this as
```sh
TOOLCHAIN_PATH=/path/to/toolchain ./meson_init.sh
```
but I'm ok with this as-is.

Reply:
I like the export because it's sticky.